test/unit/StakingProducts/getEffectiveWeight.js

```diff
@@ -82,7 +82,6 @@ describe('getEffectiveWeight', function () {
     expect(effectiveWeight).to.equal(100);
   });
 
-  // TODO: call directly into the staking products contract instead of calling SPMockCover
```

Comment:
where is the actual todo resolved?

Reply:
I pushed the change for this, now it's calling stakingPool (not a mock) instead of using SPMockCover as proxy